etc/config/glsl.defaults.properties

```diff
@@ -0,0 +1,12 @@
+compilers=&glslang
```

Comment:
These defaults aren't used by Compiler Explorer itself, so to get this on the live site we'll need to make a `glsl.amazon.properties`. I'll test locally, merge and add it.

Reply:
> I'll test locally, merge and add it

Thanks!!!